CMakeLists.txt

```diff
@@ -5,23 +5,20 @@
 ##############################################################################
 
 cmake_minimum_required (VERSION 3.4)
-project (HELICS)
+cmake_policy(VERSION 3.4)
+
+project (HELICS VERSION 1.2.1)
 
 #-----------------------------------------------------------------------------
 # HELICS Version number
 #-----------------------------------------------------------------------------
-set (HELICS_VERSION_MAJOR 1)
-set (HELICS_VERSION_MINOR 2)
-set (HELICS_VERSION_PATCH 1)
 set (HELICS_VERSION_BUILD )
 set (HELICS_DATE "06-30-18")
 
-if (NOT HELICS_VERSION_BUILD)
-	set (HELICS_VERSION "${HELICS_VERSION_MAJOR}.${HELICS_VERSION_MINOR}.${HELICS_VERSION_PATCH}")
-	set (HELICS_VERSION_UNDERSCORE "${HELICS_VERSION_MAJOR}_${HELICS_VERSION_MINOR}_${HELICS_VERSION_PATCH}")
-else()
-	set (HELICS_VERSION "${HELICS_VERSION_MAJOR}.${HELICS_VERSION_MINOR}.${HELICS_VERSION_PATCH}-${HELICS_VERSION_BUILD}")
-	set (HELICS_VERSION_UNDERSCORE "${HELICS_VERSION_MAJOR}_${HELICS_VERSION_MINOR}_${HELICS_VERSION_PATCH}-${HELICS_VERSION_BUILD}")
+set (HELICS_VERSION_UNDERSCORE "${HELICS_VERSION_MAJOR}_${HELICS_VERSION_MINOR}_${HELICS_VERSION_PATCH}")
```

Comment:
Can these be non-numeric?  specifically can HELICS_VERSION_TWEAK be non-numeric?

Reply:
Unfortunately no, it is all numeric.